config/db.test.js

```diff
@@ -0,0 +1,63 @@
+import mongoose from "mongoose";
+import connectDB from "./db";
+
+// Mock colors
+jest.mock("colors", () => {
+    const proto = String.prototype;
+    ["bgMagenta", "bgRed", "white"].forEach((k) => {
+        if (!Object.getOwnPropertyDescriptor(proto, k)) {
+            Object.defineProperty(proto, k, {
+                get() {
+                    return this.toString();
+                },
+                configurable: true,
+            });
+        }
+    });
+    return { __esModule: true, default: {} };
+});
```

Comment:
i dont think we need such heavy implementation for mocking "colors", maybe we can just do this instead?

```suggestion
jest.mock("colors")
```